### ISSUE TITLE
Impors with binary crypto dependencies fail

### ISSUE THREAD

**Issue by alytle** [dliggat/local-lambda-toolkit#9]
I built a Lambda which used `pysftp`, which uses the cryptography library. Since I built it on my Mac, I get errors about ELF versions when it tries to load the .so in the Lambda.

I used https://medium.com/i-like-big-data-and-i-cannot-lie/how-to-create-an-aws-lambda-python-3-6-deployment-package-using-docker-d0e847207dd6 to get around this, which I feel like we could add to the workflow to build the package (optionally?). 

Note that this procedure has an error (It recommends zip * instead of zip .), which if you do that results in this error: https://github.com/snowflakedb/examples/issues/4

**Comment by dgomesbr** [comment]
Hey @alytle, with the announcement of layers, this is a great candidate to serve as a layer for lambda to get that.

**Comment by alytle** [comment]
Yeah, I thought about that. I'm not sure how we would implement that in this current model though. We wouldn't always want our dependencies in a Layer? Or maybe that's not such a bad idea. Create a Layer with all libraries and the Lambda code is just our handler?

**Comment by dliggat** [comment]
Will explore the layers option for this... initially inclined towards that pathway
